<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Shashwat" ignored="0" name="Test-Online_Enrollment-CA" tests="2" failures="1" timestamp="2021-09-25T00:26:21 IST" time="50.584" errors="0">
  <testcase name="createGuestOrder" time="14.072" classname="framework.test.OnlineEnrollment_CA_PartyLite">
    <skipped/>
  </testcase> <!-- createGuestOrder -->
  <testcase name="createGuestOrder" time="13.851" classname="framework.test.OnlineEnrollment_CA_PartyLite">
    <failure type="java.lang.RuntimeException">
      <![CDATA[java.lang.RuntimeException
at framework.utils.Reports.reportStep(Reports.java:72)
at framework.utils.Reports.reportStep(Reports.java:82)
at framework.test.OnlineEnrollment_CA_PartyLite.createGuestOrder(OnlineEnrollment_CA_PartyLite.java:105)
at java.util.ArrayList.forEach(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- createGuestOrder -->
</testsuite> <!-- Test-Online_Enrollment-CA -->
